.travis.yml

```diff
@@ -34,6 +34,16 @@ jobs:
         - DEPLOY="true"
         - USE_MAVEN_REPO="true"
 
+    # sevntu (openjdk11)
+    - jdk: openjdk11
+      env:
+        - DESC="sevntu"
+        - CMD1="mvn -e --no-transfer-progress clean compile antrun:run@ant-phase-verify-sevntu"
+        - CMD2=" -Psevntu"
+        - CMD="$CMD1$CMD2"
```

Comment:
Line was too long, so I had to split the 2.

Reply:
Can we remove "clean" ?
Host is always clean after clone

Reply:
Done.